src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1387,29 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+        // Compute forward kinematics
+
+        for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_model.getNrOfLinks()); idx++)
+        {
+            IndexRange sensorRange = this->getRangeLinkSensorVariable(NET_EXT_WRENCH_SENSOR, idx);
+            IndexRange netExtWrenchRange = this->getRangeLinkVariable(NET_EXT_WRENCH,idx);
+
+            // Get link to base rotation
+            matrixYElements.addSubMatrix(sensorRange.offset,
+                                         netExtWrenchRange.offset,
+                                         base_H_m_links.at(idx).asAdjointTransformWrench());
```

Comment:
This is fishy. the measurement has 3 elements. but this matrix has 6 rows.

Reply:
Updated in https://github.com/robotology/idyntree/pull/571/commits/b11ff9210cf96bb9a6507d214b140a26c963b6d6
@traversaro